package.json

```
@@ -5,6 +5,7 @@
     "demo-app": "cd packages/browser; npm run demo-app",
     "bootstrap": "lerna bootstrap",
     "build": "lerna run build",
+    "clean": "rm -rf packages/*/dist/ packages/*/node_modules/",
```

No need to use the shorthand in a command, and `-f` shouldn't actually be necessary (and thus is probably best left out).

```suggestion
    "clean": "rm --recursive packages/*/dist/ packages/*/node_modules/",
```

---

Fixed in bbf7d447ecb4e6ee1b180e8cf7d839c3c991e9b9

---

This is funny - I did actually consider saying basically the same thing as Vinnie here. In general I totally agree, but I thought that in Linux-land, `rm -rf` is soooo common, that actually using `--recursive` might actually be making this **_less_** readable, rather than more readable :) ! Anyway, I'm totally fine with the change - it just struck me as funny.